code/modules/mob/living/carbon/xenomorph/castes/spitter/castedatum_spitter.dm

```diff
@@ -38,7 +38,7 @@
 
 	// *** Ranged Attack *** //
 	spit_delay = 0.8 SECONDS
-	spit_types = list(/datum/ammo/xeno/acid/medium) //Gotta give them their own version of heavy acid; kludgy but necessary as 100 plasma is way too costly.
+	spit_types = list(/datum/ammo/xeno/acid/medium, /datum/ammo/xeno/toxin/redspit) //Gotta give them their own version of heavy acid; kludgy but necessary as 100 plasma is way too costly.
```

Comment:
absolutely not unless this is promordial and even then i am on teh fence about it